### PR TITLE
Added non-clickable text component

### PR DESCRIPTION
**Pull request by AgusArdizzone**
Hi!

Just like the title says, i made a non-clickable text component (based on clock component).
Although it was a simple task, i'm sure there's room for improvements since i'm not a C# dev 😅

### CODE REVIEW

File: GlazeWM.Bar/Components/TextComponentViewModel.cs [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+      
+      // Update the displayed time every second.
+      // This is useless for this component
+      // since it's a fixed text.
+      var updateInterval = TimeSpan.FromSeconds(1);
+
+      Observable.Interval(updateInterval)
+        .Subscribe(_ => OnPropertyChanged(nameof(Text)));
+    }
```
Comment (lars-berger) [inline GlazeWM.Bar/Components/TextComponentViewModel.cs:29]: No need to call `OnPropertyChanged`, something like this should work fine:

```csharp
public class TextComponentViewModel : ComponentViewModel
{
  private TextComponentConfig _config => _componentConfig as TextComponentConfig;

  public string Text => _config.Text;

  public TextComponentViewModel(BarViewModel parentViewModel, TextComponentConfig config)
    : base(parentViewModel, config)
  {
  }
}
```

File: GlazeWM.Bar/BarViewModel.cs [hunk 2]
```diff
@@ -44,6 +44,7 @@ private List<ComponentViewModel> CreateComponentViewModels(List<BarComponentConf
       {
         WorkspacesComponentConfig wcc => new WorkspacesComponentViewModel(this, wcc),
         ClockComponentConfig ccc => new ClockComponentViewModel(this, ccc),
+        TextComponentConfig tcc => new TextComponentViewModel(this,tcc),
```
Comment (lars-berger) [inline GlazeWM.Bar/BarViewModel.cs:47]: add space after `this,`

**Review by lars-berger — CHANGES_REQUESTED** [review]

**Comment by lars-berger** [comment]
Awesome, thank you 👍 Just have some minor corrections, but otherwise looks good

**Comment by AgusArdizzone** [comment]
I uploaded the requested changes 😄

**Comment by lars-berger** [comment]
Great, looks good 👌

**Comment by lars-berger** [comment]
Took a while longer than expected, but this is now in the [latest release](https://github.com/lars-berger/GlazeWM/releases) 👍

**Pull request merged**